README.md

```diff
@@ -117,8 +117,8 @@ For APIs of other older versions, head to Wiki.
 [logo-url]: https://tarunbatra.github.io/password-validator
 [npm-image]: https://img.shields.io/npm/v/password-validator.svg?style=flat-square
 [npm-url]: https://www.npmjs.com/package/password-validator
-[travis-image]:https://img.shields.io/travis/tarunbatra/password-validator.svg?logo=travis&style=flat-square
-[travis-url]:https://travis-ci.org/tarunbatra/password-validator
 [downloads-image]: https://img.shields.io/npm/dt/password-validator.svg?style=flat-square
 [codecov-url]: https://codecov.io/gh/tarunbatra/password-validator
 [codecov-image]: https://img.shields.io/codecov/c/gh/tarunbatra/password-validator?logo=codecov&style=flat-square
+[gh_action-url]: https://github.com/tarunbatra/password-validator/actions/workflows/build.yml/badge.svg
```

Comment:
I feel like we should use shields.io badges here, to style it consistently with other badges, but it is not a big thing and I can do it after the merge as well.

Reply:
Makes sense, have added the badge also. 🚀